resources/dynamodb_tables.go

```diff
@@ -0,0 +1,677 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func DynamodbTables() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_dynamodb_tables",
+		Description: "Represents the properties of a table.",
```

Comment:
maybe gives this a better description, you can override with description attribute